todoman/confspec.ini

```diff
@@ -14,6 +14,13 @@ color = option('always', 'auto', 'never', default='auto')
 # this option is not specified the system locale's is used.
 date_format = string(default='%x')
 
+# The date format used both for displaying times, and parsing input times.
+# Default is `%H:%M`.
```

Comment:
There's no need to clarify this default due to how the doc is rendered (the one from below will be picked up and shown already).